hnn_core/tests/test_gui.py

```diff
@@ -90,7 +90,8 @@ def check_drive(drive1, drive2, keys):
         for cell_type, bias_params in bias_dict.items():
             check_items(bias_params,
                         net2.external_biases[bias_name][cell_type],
-                        ignore_keys=['tstop'],
+                        # OK to ignore section here?
+                        ignore_keys=['tstop', 'section'],
```

Comment:
We'll have to update the GUI for this tonic bias API change. I think what would make the most sense is creating a side branch for "multiple_tonics". We'll merge this PR into that branch and then add a second PR with the GUI changes. Once both are combined and working, we can merge it into main. Thoughts @asoplata @jasmainak @ntolley @dylansdaniels ?

Reply:
is the multiple tonic feature required in the GUI? In any case, I think it could be a follow-up PR ... wouldn't make side branches, it makes for more complicated workflows

Reply:
If someone tired to load a network configuration file with multiple tonics it wouldn't work. I'm not exactly sure what would happen but I think there are checks to make sure that only one tonic is allowed and it must be named a certain way.